brightray/browser/url_request_context_getter.cc

```diff
@@ -280,8 +282,7 @@ net::URLRequestContext* URLRequestContextGetter::GetURLRequestContext() {
 
     std::unique_ptr<net::TransportSecurityState> transport_security_state =
         base::WrapUnique(new net::TransportSecurityState);
-    transport_security_state->SetRequireCTDelegate(
-        delegate_->GetRequireCTDelegate());
+    transport_security_state->SetRequireCTDelegate(ct_delegate_.get());
     storage_->set_transport_security_state(std::move(transport_security_state));
     storage_->set_cert_verifier(delegate_->CreateCertVerifier());
```

Comment:
Its better to pass the `ct_delegate` as a parameter of `CreateCertVerifier`, can avoid having a `GetRequireCTDelegate` getter.

Reply:
:+1: